examples/private_connection/main.tf

```diff
@@ -47,41 +47,6 @@ resource "google_compute_subnetwork" "subnet" {
   ip_cidr_range            = "10.127.0.0/20"
   network                  = google_compute_network.network.self_link
   private_ip_google_access = true
-  log_config {
-    aggregation_interval = "INTERVAL_10_MIN"
-    flow_sampling        = 1
-    metadata             = "INCLUDE_ALL_METADATA"
-  }
-}
-
-# Router to control egress connectivity
-resource "google_compute_router" "forseti-router" {
-  count   = var.block_egress ? 0 : 1
-  name    = "forseti-network-router"
-  network = google_compute_network.network.name
-  region  = var.region
-}
-
-resource "google_compute_router_nat" "forseti-nat" {
```

Comment:
Why is this no longer needed?

Reply:
NAT was removed due rushed actions and is restored. logs were removed since they aren't essential for the example but increase cost of deployment